src/components/processes-configuration.js

```diff
@@ -312,12 +252,12 @@ const ProcessesContainer = ({ handleProcessesChanged, initialConfigs }) => {
                     <FormattedMessage id="tsos" />
                 </Grid>
             </Grid>
-            {currentProcesses.map((process, index) => (
+            {currentConfigs.map((process, index) => (
                 <Grid
                     container
                     spacing={1}
                     className={classes.addNewTso}
-                    key={process.id}
+                    key={'processEdit' + index}
```

Comment:
why change this ? The previous one allowed for deletes to correctly map the js data to the dom structure without resetting everything